app/views/comments/_comment.html.erb

```diff
@@ -0,0 +1,5 @@
+<form action="<%= post_comments_path(post) %>" method="post">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <textarea name="comment[text]" rows="6" cols="20"></textarea>
+  <input type="submit" value="Submit">
+</form>
```

Comment:
- [ ] There should be a form in the views/comments/new.html.erb file which accepts input for creating comments.

Reply:
Hey @algerina. I think I am mentioned here by mistake. Kindly confirm. Because I am waiting for my review.